### PR TITLE
Fixes #733: Add a Horizontal Rule brick

### LINKED ISSUE #733: Add a Horizontal Rule brick
It allows us to easily separate content. A good use case can be found in the survey.

profile 4 1x

### PR DESCRIPTION
**Pull request by gesa**
Add a brick page:
screen shot 2015-01-20 at 18 42 11

The edit page is *maybe* a little weird? Not sure.
screen shot 2015-01-20 at 18 43 00

But I think it looks good on the make/:id page!
screen shot 2015-01-20 at 18 43 36

### CODE REVIEW

File: blocks/separator/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    className: 'separator',
+    template: require('./index.html'),
+    data: {
+        name: 'Separator',
```
Comment (gesa) [inline blocks/separator/index.js]: There's some kind of weird whitespace here, but I was trying to keep the same code style as other blocks.

**Comment by k88hudson** [comment]
@gesa the HR should be `@mediumGrey` and flush with the side of the text

File: blocks/separator/index.less [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.separator {
+  hr {
+    color: @darkGrey;
+    margin: 15px auto;
+  }
```
Comment (k88hudson) [inline blocks/separator/index.less]: needs border: `1px solid @mediumGrey`  since hrs have a weird border on them by default

**Comment by k88hudson** [comment]
Looking good just a couple of things

**Comment by k88hudson** [comment]
@gesa hey just making sure you saw my comments?

**Comment by gesa** [comment]
Yes! Sorry, was knee-deep in privacy day stuff. I should be wrapping this today.

**Comment by k88hudson** [comment]
no prob just making sure ^_^

**Comment by gesa** [comment]
All set @k88hudson!

File: blocks/separator/index.less [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.separator {
+  hr {
+    border-bottom: 1px solid @mediumGrey;
```
Comment (k88hudson) [inline blocks/separator/index.less]: arg, you will also need to add `border: none` above this

**Comment by k88hudson** [comment]
One more small css nit;

I noticed that it's weird that you can open the editor for this when there are no editable attributes. Either add one (like color maybe, to set the color of the `hr`) or we need to have a way to check if a block has editable attributes (if not, we should not open the editor)

**Comment by gesa** [comment]
TIL our parser cannot handle self-closing tags. For some reason. DON'T DO IT! Anyway now you can change the color of the `<hr/>`

**Comment by thisandagain** [comment]
Anything else needed to close this out @gesa @k88hudson ?

**Comment by k88hudson** [comment]
R+

**Comment by k88hudson** [comment]
merge away

**Pull request merged**